kernel/constr.ml

```diff
@@ -1295,6 +1306,7 @@ let rec hash t =
        combinesmall 14 (combine (hash_term_array bl) (hash_term_array tl))
     | Meta n -> combinesmall 15 n
     | Rel n -> combinesmall 16 n
+    | Int i -> combinesmall 17 (Uint63.hash i)
```

Comment:
`Proj` is already using 17

(for some reason we have 2 hashing functions on constr?)

Reply:
Well spotted, fixed.